sds_data_manager/lambda_code/requirements.txt

```diff
@@ -95,3 +95,5 @@ six==1.16.0 ; python_version >= "3.9" and python_version < "4" \
 urllib3==1.26.16 ; python_version >= "3.9" and python_version < "4" \
     --hash=sha256:8d36afa7616d8ab714608411b4a3b13e58f463aee519024578e062e141dce20f \
     --hash=sha256:8f135f6502756bde6b2a9b28989df5fbe87c9970cecaa69041edcce7f0589b14
+requests-aws4auth==1.2.3 ; python_version >= "3.9" and python_version < "4" \
```

Comment:
I meant to comment on past PR but didn't get to it. What is this requirements.txt used for?

Reply:
The requirements.txt is provides those libraries for the lambdas to access. Although it sounds like I might be able to just put that in the pyproject.toml

Reply:
This requirements.txt file is used for pip installs. It is automatically generated in the pre-commit steps, and should not be updated by hand. It is created from the dependencies specified in pyproject.toml. Let me know if you have other questions about it!